skimage/filters/ridges.py

```diff
@@ -286,6 +298,14 @@ def sato(image, sigmas=range(1, 10, 2), black_ridges=True):
     if np.any(sigmas < 0.0):
         raise ValueError('Sigma values less than zero are not valid')
```

Comment:
Hey @rfezzani,
since you're still here, could you change this `ValueError` to something more friendly? I would go for `'Sigma values should be greater than zero.'`

Reply:
Shouldn't `sigma` be strictly positive?

Reply:
No, I checked, `sigma = 0` is OK, sorry :pray:

Reply:
Could you change the message for `'Sigma values should be equal to or greater than zero.'`, then? I don't like the _tone_ of that message, actually. :blush:

Reply:
I already modified the message as you suggested in the new `_check_sigmas` function :-)
What about `Sigma values should be positive.`?

Reply:
Positive means that the zero is not included :blush:
Since zero can be used, something more accurate would be non-negative, in this case. However, what non-negative means is not that clear – the user could think that positive and non-negative mean the same thing, but they don't.
Hence, `'Sigma values should be equal to or greater than zero.'` :smile:

Reply:
No objection :-).